bin/docker.preinit

```diff
@@ -34,6 +34,7 @@ fi
 su $FMS_DB_USER -c "${FMS_ROOT}/bin/docker-cobrand"
 
 # Ensure things are up to date - schema, CSS, etc
+su $FMS_DB_USER -c "${FMS_ROOT}/bin/install_packages docker"
```

Comment:
This is going to need sudo, which is fine as there is already a sudo rule in place to allow it.

Reply:
Missed that the first time around, sorry!

Reply:
Have replaced whole line with `${FMS_ROOT}/bin/install_packages docker` assuming that'll work :)

Reply:
👍 test build from this commit for both Docker image and AMI look good.